primary/public-src/ts/script-editform.ts

```diff
@@ -0,0 +1,217 @@
+import type { AbsoluteValueOperation, CompareOperation, DivideOperation, LogOperation, MinMaxOperation, MultiplyOperation, MultiselectOperation, operand, SumOperation } from 'scoutradioz-types';
+
+type LayoutEdit = import('scoutradioz-types').LayoutEdit;
+type StringDict = import('scoutradioz-types').StringDict;
+type DerivedLayout = import('scoutradioz-types').DerivedLayout;
+
+// consolesdflkjd
```

Comment:
Does **consolesdflkjd** mean something I'm not aware of?

Reply:
oops, typo!